src/callbacks_step/analysis_dg2d.jl

```diff
@@ -149,6 +149,50 @@ function calc_error_norms(func, u, t, analyzer,
     return l2_error, linf_error
 end
 
+function calc_error_norms(func, u, t, analyzer,
+                          mesh, equations,
+                          initial_condition, solver::FV, cache, cache_analysis)
```

Comment:
This is probably only valid for the mesh type `T8codeMesh`. TODO: Add dispatching